### ISSUE TITLE
Overfitting

### ISSUE THREAD

**Issue by rmattson1008** [rmattson1008/face-less#11]
Our model is overfitting to the train set before it can get a good score on the test set. One way to fight this is to use a smaller model (less parameters = less ability to just memorize data points). I suggest we at least use vgg-16 if not squeezenet (both available already through pytorch)
